lib/index.js

```diff
@@ -13,16 +13,16 @@ if (require.main === module) {
 
     load (configPath) {
       this.configPath = configPath;
+      process.env.EG_CONFIG_DIR =
+      this.configPath || process.env.EG_CONFIG_DIR;
+      this.config = require('./config'); // this is to init config before loading servers and plugins
```

Comment:
I believe this was in `run()` instead of `load()` due to a timing issue with loading the config.  That may be obsolete by now, but it's worth pointing out.

Reply:
Timing issue? Do you recall an example in particular?

Reply:
Don't quite remember now.  I know our config singleton is a fragile beast.  If all the tests are passing, it's probably no longer an issue.